### ISSUE TITLE
Adding users requires setting 'Verify' flag

### ISSUE THREAD

**Issue by jschirrmacher** [RocketChat/Rocket.Chat#15798]
### Description:

When an administrator adds a new user to the chat and leaves the `verify` switch off, the new user gets an email containing username and password, but cannot login until the administrator switches the `verify` flag on.

### Steps to reproduce:

1. As administrator go to 'Administration / Users' and click on '+' symbol in right sidebar.
2. Fill in necessary user fields, but keep 'Verified' switch off.
3. Save the user.

### Expected behavior:

The new user should be able to log in with the credentials from the email.

### Actual behavior:

Login is only possible if administrator switches 'Verified' on.

### Server Setup Information:

- Version of Rocket.Chat Server: 1.3.2
- Operating System: Linux
- Deployment Method: tar
- Number of Running Instances: 2
- DB Replicaset Oplog: Enabled
- NodeJS Version: v8.15.1
- MongoDB Version: 4.0.5

**Comment by reetp** [comment]
> Please see our guide for opening issues: https://rocket.chat/docs/contributing/reporting-issues

**Comment by reetp** [comment]
Ha - just saw your PR so I guess the answer is it is the same?

**Comment by jschirrmacher** [comment]
Yes, in fact, it is.

**Comment by pierre-lehnen-rc** [comment]
I tested this on 3.1.0 and the login seemed to be working fine (except for the bug described on issue #17136, which is a bit different), can you check if your issue is fixed now?

**Comment by sampaiodiego** [comment]
there is a setting called "Email Verification" under Admin > Accounts > Registration, that enforces the requirement of a verified email to allow an user to log in.

so if you have that setting on, I don't see an issue here.. except the fact the UI _could_ be improved in this case.

**Comment by jschirrmacher** [comment]
Thanks, @pierre-lehnen-rc  and @sampaiodiego. I checked the behavior after updating  to 3.1.1 and can confirm that it depends on the setting you mentioned.

While this helps me with my original problem, I don't think that this only an "improvement of the UI problem". Actually, with allowing only users with verified email addresses, it shouldn't be possible to create a user without having the "verify" flag set to off, or at least this shouldn't be turned off by default, because this is highly error-prone (like I experienced).

**Comment by rodrigok** [comment]
I think we should send the verification email to the informed email is admin didn't set as verified at creating time. What do you think?

**Comment by jschirrmacher** [comment]
Yes, this sounds reasonable. If only users with verified e-mail addresses are allowed and the administrator doesn't set the 'verified' flag, the user should should receive an e-mail with a verification link and if they click it, the user should be allowed to use the account.
